test/test_utils/ec2.py

```diff
@@ -435,42 +437,120 @@ def execute_ec2_inference_test(connection, ecr_uri, test_cmd, region=DEFAULT_REG
     )
 
 
-def execute_ec2_training_performance_test(connection, ecr_uri, test_cmd, region=DEFAULT_REGION):
+def execute_ec2_training_performance_test(connection, ecr_uri, test_cmd, region=DEFAULT_REGION,
+                                          post_process=None, data_source="", threshold=None):
     docker_cmd = "nvidia-docker" if "gpu" in ecr_uri else "docker"
     container_test_local_dir = os.path.join("$HOME", "container_tests")
 
+    timestamp = time.strftime('%Y-%m-%d-%H-%M-%S')
+    log_name = f"{data_source}_results_{os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')}_{timestamp}.txt"
```

Comment:
Please ensure there are no conflicts in log file names caused by multiple tests running in parallel. Rename the file to something much more specific, if needed.

Reply:
ack. should have no conflicts